[2024-05-21T14:38:14.374+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T14:38:14.399+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2022-06-27T00:00:00+00:00 [queued]>
[2024-05-21T14:38:14.410+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2022-06-27T00:00:00+00:00 [queued]>
[2024-05-21T14:38:14.411+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-21T14:38:14.431+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): generate_random_numbers> on 2022-06-27 00:00:00+00:00
[2024-05-21T14:38:14.437+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=7228) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T14:38:14.439+0000] {standard_task_runner.py:63} INFO - Started process 7237 to run task
[2024-05-21T14:38:14.440+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'statistical_calculations_dag', 'generate_random_numbers', 'scheduled__2022-06-27T00:00:00+00:00', '--job-id', '2288', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmp53qmikcr']
[2024-05-21T14:38:14.441+0000] {standard_task_runner.py:91} INFO - Job 2288: Subtask generate_random_numbers
[2024-05-21T14:38:14.494+0000] {task_command.py:426} INFO - Running <TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2022-06-27T00:00:00+00:00 [running]> on host c420abfdc210
[2024-05-21T14:38:14.601+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='statistical_calculations_dag' AIRFLOW_CTX_TASK_ID='generate_random_numbers' AIRFLOW_CTX_EXECUTION_DATE='2022-06-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-06-27T00:00:00+00:00'
[2024-05-21T14:38:14.602+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T14:38:14.629+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T14:38:14.630+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T14:38:14.682+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=statistical_calculations_dag, task_id=generate_random_numbers, run_id=scheduled__2022-06-27T00:00:00+00:00, execution_date=20220627T000000, start_date=20240521T143814, end_date=20240521T143814
[2024-05-21T14:38:14.734+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T14:38:14.766+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-21T14:38:14.770+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
